src/App.jsx

```diff
@@ -14,57 +16,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoods, setSelectedGood] = useState(8);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGoods === -1 ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {goods[selectedGoods]} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood(-1)}
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => {
+            const valueBool = index === selectedGoods;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                data-cy="Good"
+                className={cn({ 'has-background-success-light': valueBool })}
+              >
+                <td>
+                  <button
+                    data-cy={valueBool ? 'RemoveButton' : 'AddButton'}
+                    type="button"
+                    className={cn({ 'is-info': valueBool, button: true })}
+                    onClick={() => {
+                      return valueBool
+                        ? setSelectedGood(-1)
+                        : setSelectedGood(index);
+                    }}
```

Comment:
Suggestion: change `valueBool` to `isSelected` then rewrite to `setSelectedGood(isSelected? -1 : index)` for improved readability